src/wallet/rpcwallet.cpp

```diff
@@ -4489,7 +4489,20 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             "    }, ... ]\n"
             "3. minconf               (numeric, optional, default=1) Only use funds confirmed at least this many times.\n"
             "4. fee                   (numeric, optional, default=" + strprintf("%s", FormatMoney(DEFAULT_FEE)) + ") The fee amount to attach to this transaction.\n"
-            "5. allowRevealedAmounts  (bool, optional, default=false) Permit cross-shielded-pool transfers, which will publicly reveal the amount(s) crossing pool boundaries.\n"
+            "5. privacyPolicy         (string, optional, default=\"LegacyCompat\") Policy for what information leakage is acceptable.\n"
+            "                         One of the following strings:\n"
+            "                               - \"FullPrivacy\": Only allow fully-shielded transactions (involving a single shielded pool).\n"
+            "                               - \"LegacyCompat\": If the transaction involves any UAs, this is equivalent to \"FullPrivacy\".\n"
+            "                                 Otherwise, this is equivalent to \"AllowFullyTransparent\".\n"
+            "                               - \"AllowRevealedAmounts\": Allow funds to cross between shielded pools.\n"
+            "                               - \"AllowRevealedRecipients\": Allow transparent recipients. Implies \"AllowRevealedAmounts\".\n"
```

Comment:
"Implies" is ambiguous. (Does it imply the same security properties? No, the opposite! I think this is not immediately obvious, and the intended meaning may be a lot clearer to us than it is to users or even wallet implementors.)
```suggestion
            "                               - \"AllowRevealedRecipients\": Allow transparent recipients. This also implies revealing information described under \"AllowRevealedAmounts\".\n"
```